sound/soc/codecs/rt1308-sdw.c

```diff
@@ -693,7 +693,7 @@ static int rt1308_sdw_probe(struct sdw_slave *slave,
 }
 
 static const struct sdw_device_id rt1308_id[] = {
-	SDW_SLAVE_ENTRY(0x025d, 0x1308, 0),
+	SDW_SLAVE_ENTRY_EXT(0x025d, 0x1308, 0x2, 0, 0),
```

Comment:
why is version hexadecimal? Is it supposed to have some special meaning? It isn't just a counter?

Reply:
The version is encoded on 4 bits, that's what I was trying to convey

Reply:
well, what makes more sense: "version 0xb" or "version 11?"

Reply:
0xb, that's what we see in the devID registers